<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterWindow</class>
 <widget class="QMainWindow" name="CharacterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Character</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="EditCharacter" name="character_def">
       <attribute name="title">
        <string>Perso</string>
       </attribute>
      </widget>
      <widget class="EditAttakView" name="edit_atk_tab">
       <attribute name="title">
        <string>Attaque</string>
       </attribute>
      </widget>
      <widget class="StuffsView" name="edit_stuff_view">
       <attribute name="title">
        <string>Add stuff</string>
       </attribute>
      </widget>
      <widget class="EquipOnBodyView" name="use_stuff_view">
       <attribute name="title">
        <string>Use stuff</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="CompBigSecondaryButton" name="pushButton">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CompBigSecondaryButton" name="apply_pushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditAttakView</class>
   <extends>QWidget</extends>
   <header location="global">editattakview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditCharacter</class>
   <extends>QWidget</extends>
   <header location="global">editcharacter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StuffsView</class>
   <extends>QWidget</extends>
   <header location="global">stuffsview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EquipOnBodyView</class>
   <extends>QWidget</extends>
   <header location="global">equiponbodyview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CompBigSecondaryButton</class>
   <extends>QPushButton</extends>
   <header location="global">CompButtons.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
